config/bundle-config.js

```diff
@@ -34,6 +34,15 @@ const DEFAULT_DATA_SOURCES = exports.DEFAULT_DATA_SOURCES = [
       'neuroglancer/async_computation/decode_gzip',
     ],
   },
+  {
+    source: 'neuroglancer/datasource/deepzoom',
+    asyncComputation: [
+      'neuroglancer/async_computation/decode_jpeg',
+      'neuroglancer/async_computation/decode_gzip',
+      'neuroglancer/async_computation/decode_compresso',
```

Comment:
compresso can be removed, that is not relevant to deepzoom

Reply:
Yes. I haven't re-visited this file besides the intial addition and commenting-uncommenting stuff en masse.

Reply:
Done.